github/ci/prow-deploy/files/jobs/kubevirt/hyperconverged-cluster-operator/hyperconverged-cluster-operator-presubmits-1.14.yaml

```diff
@@ -0,0 +1,74 @@
+presubmits:
+  kubevirt/hyperconverged-cluster-operator:
+  - name: pull-hyperconverged-cluster-operator-e2e-k8s-1.30
+    branches:
```

Comment:
so it's exactly the same for both k8s-1.30 and k8s-1.31?

Reply:
no - the K8s versions are not the same